tests/unit/nulls.c

```diff
@@ -146,6 +146,23 @@ int main(int argc, char *argv[])
   CHECKED_QUERY(querybuf);
 
 #ifdef TEST_PREPARED_STATEMENTS
+  strcpy(querybuf, "SELECT NULL");
+  sth = drizzle_stmt_prepare(con, querybuf, strlen(querybuf), &driz_ret);
+
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, preparing \"%s\"",
+             drizzle_strerror(driz_ret), drizzle_error(con), querybuf);
+
+  driz_ret = drizzle_stmt_execute(sth);
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, executing \"%s\"",
+    drizzle_strerror(driz_ret), drizzle_error(con), querybuf);
```

Comment:
Why the different indentation on this line (as against in the `ASSERT_EQ_()` statements before and after)?